editor.planx.uk/src/@planx/components/shared/Preview/NavigateToPublishedButton.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from "@mui/material/Link";
+import Typography from "@mui/material/Typography";
+import React from "react";
+import { useNavigation } from "react-navi";
+
+import { InnerContainer } from "./SaveResumeButton";
+
+const NavigateToPublishedButton: React.FC = () => {
+  const navigation = useNavigation();
+
+  const testEnvironment = window.location.pathname.endsWith("/draft")
+    ? "/draft"
+    : "/preview";
+
+  const editorLink = window.location.pathname.replace(
+    testEnvironment,
+    "/published",
+  ) as `/${string}`;
+
+  const handleClick = () => {
+    navigation.navigate(editorLink);
+  };
```

Comment:
```suggestion
  const { navigate } = useNavigation();

  const handleClick = () => {
    navigate("published");
  };
```

Reply:
I think this would do the same thing but much simpler - we just get react-navi to take us to a new route - it's always "published".